src/styles/blocks/card.scss

```diff
@@ -0,0 +1,100 @@
+.card {
+  width: $cardWidth;
+  padding: 32px 16px 16px;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__rate-container {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    margin-bottom: 24px;
+  }
+
+  &__price-container {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    margin-bottom: 16px;
+  }
+
+  &__image {
+    display: block;
+
+    width: 160px;
+    height: 134px;
+
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+
+    width: 166px;
+    height: 36px;
```

Comment:
Does title really need width and height? Would anything change if you remove them?

Reply:
+1
You don't need to hardcode your title. 
h tag is a block, so it will take 100% of the width. 
About height, use line-height for this purpose, for example, what will be in case when the text will wrap to the second line?

Reply:
Good point,done!